src/ApparentHorizons/Tags.hpp

```diff
@@ -685,6 +685,35 @@ struct DimensionfulSpinMagnitudeCompute : DimensionfulSpinMagnitude,
                  StrahlkorperTags::Strahlkorper<Frame>, AreaElement<Frame>>;
 };
 
+/// The dimensionful spin angular momentum vector.
+struct DimensionfulSpinVector : db::SimpleTag {
+  using type = std::array<double, 3>;
+};
+
+/// Computes the dimensionful spin angular momentum vector.
+template <typename Frame>
+struct DimensionfulSpinVectorCompute : DimensionfulSpinVector, db::ComputeTag {
+  using base = DimensionfulSpinVector;
+  using return_type = std::array<double, 3>;
+  static std::array<double, 3> function(
```

Comment:
This should return by reference.

Reply:
@wthrowe, should I make a test for all the compute tags that I have merged into Tags.hpp or should I only make this test for the DimensionfulSpinVectorTag?

Reply:
They should all have tests, but you don't have to do the others in this PR.

Reply:
It looks like this test passes now, there is one issue, the Radius tag type does not match the function call types. The type is a `DataVector` and the functions call for a `Scalar<DataVector>`. Is that something that needs to be changed and updated for the Radius tag or is it okay to change all the function calls to take a DataVector instead for the radius arguments? @wthrowe

Reply:
It should probably be a `Scalar<DataVector>`, but let's get @markscheel's input on this.

Reply:
I would be fine if RadiusCompute (and Radius) were changed to `Scalar<DataVector>`.  Other ComputeItems depend on `RadiusCompute` and they should be changed too.  Is the idea to do that change in a subsequent PR?

Reply:
Thank you @markscheel, Should I just implement this change in the current PR? @wthrowe

Reply:
Let's do it as a separate commit here.  It will be needed to make the test pass.

Reply:
The test is now passing and the RadiusCompute tag was changed to a type `Scalar<DataVector>`. @wthrowe